PoissonRegression.ipynb

```diff
@@ -0,0 +1,5371 @@
+{
```

Comment:
<p>I recommended calling `describe(chain)` above, so you can remove all of these `print` calls and manually calculating the means of each parameter.</p><p><br></p><p>You can also simply call `plot(chains)` instead of plotting `plot(chain)`, MCMCChains is pretty good about plotting multiple chains together.</p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Changed. Now, this block includes just one line: `plot(chains)`.</p>

Reply:
<p>I think that the print statements should still be included because the `describe(chains)` function gives the means. What we require is the exponentiated means to give a better sense of the relative values.</p>

Reply:
Ah, okay. Then cut `describe` entirely, and make sure the `print` calls are in a separate block from `plot`. I'd also add a line or two in the text block above describing why you're exponentiating the means.

Reply:
Used print calls instead of `describe(chains)`. Added a line explaining the reason behind using exponentiated means:
We take the exponent of the means to get a better comparison of the relative values of the coefficients.